CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Update the `dump` function to handle addresses: PR [#175](https://github.com/tact-lang/tact/pull/175)
-- Support trailing commas for struct fields and function arguments: PR [#179](https://github.com/tact-lang/tact/pull/179)
+- Support trailing commas in all comma-separated lists (struct instance fields, `initOf` arguments, `init()` parameters, inherited traits via `with`, function arguments and parameters): PR [#179](https://github.com/tact-lang/tact/pull/179) and PR [#246](https://github.com/tact-lang/tact/pull/246)
```

Comment:
Let's also support trailing commas in struct declarations

Reply:
Hmm, there aren't any — fields of structs and messages end with semi-colons `;`

Reply:
argh, you are right -- it's an inconsistency in the grammar design